lib/service/storage/elasticsearch.ts

```diff
@@ -1728,13 +1921,20 @@ class ElasticSearch extends Service {
    * @returns {Promise.<{ items, errors }>
    */
   async import(
-    index,
-    collection,
-    documents,
-    { refresh, timeout, userId = null } = {},
+    index: string,
+    collection: string,
+    documents: JSONObject[],
+    {
+      refresh,
+      timeout,
+      userId = null,
+    }: {
+      refresh?: boolean | "wait_for";
+      timeout?: string;
+      userId?: string;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      timeout,
      userId = null,
    }:  Exclude<KRequestParams, "injectKuzzleMeta" | "retryOnConflict"> = {},
```
With new `KRequestParams` type we can compose type like it, to DRY